posthog/api/feature_flag.py

```diff
@@ -105,7 +105,7 @@ class FeatureFlagViewSet(StructuredViewSetMixin, AnalyticsDestroyModelMixin, vie
     permission_classes = [IsAuthenticated, ProjectMembershipNecessaryPermissions]
 
     def get_queryset(self) -> QuerySet:
-        queryset = FeatureFlag.objects.filter(team=self.team)
```

Comment:
Why won't this allow anyone access to feature flags of all teams?

Reply:
This was a change from my last PR, when I implemented this just for `api/feature_flag`. I just reverted it here because the queryset from the parent now does this checking.